### PR TITLE
Add missing List.toSet() documentation

### PR DESCRIPTION
**Pull request by jimscarver**
### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

**Review by zsluedem — APPROVED** [review]
LGTM

File: rholang/reference_doc/data_structures/README.md [hunk 1]
```diff
@@ -114,6 +114,7 @@ Converts string to UTF-8 byte array. Returns an error when called with arguments
 #### Ordered data structure methods
 - `length()`: the number of elements in a byte array, list, or string
 - `slice(m,n)`: slice of a byte array, list, or string
+- `toSet()`: generates a Set of list elements eliminating guplicates
```
Comment (tgrospic) [inline rholang/reference_doc/data_structures/README.md]: It's a typo in `guplicates`.

Please make it clear that method is applicable to list and map structures.

**Review by tgrospic — APPROVED** [review]
LGTM

@jimscarver Thanks for the support!

**Comment by tgrospic** [comment]
bors delegate+

@jimscarver You should now be able to merge your PR with `bors merge`.

**Pull request merged**